nodes/combine-logic.js

```diff
@@ -69,27 +69,19 @@ module.exports = function (RED) {
                 topics: this.distinction === 'topic' ? topics : undefined,
                 messages: this.distinction === '_msgid' ? topics : undefined,
                 and: payloads.reduce((pv, cv) => pv && cv),
-                or: this.or(payloads),
+                or: payloads.reduce((pv, cv) => pv || cv),
```

Comment:
cleaner code, had the solution with the for-loop because it promises slightly better performance because it breaks the loop on the first truthy payload. But I think cleaner code is more important here than performance, should not matter under normal circumstances.